tests/about.spec.ts

```diff
@@ -107,14 +107,15 @@ test.describe('About Page Tests', () => {
     )
   })
 
-  test('About YouTube modal plays video', async ({ page }) => {
+  test('the YouTube modal in "about" is visible and have the correct src', async ({ page }) => {
     await page.goto('/about')
-    const videoFrame = page.frameLocator('iframe[title="YouTube video player"]')
-    const playButton = videoFrame.getByLabel('Play', { exact: true })
-    await playButton.click()
-    await playButton.waitFor({ state: 'hidden' })
-    const video = videoFrame.locator('video')
-    await video.hover()
-    await expect(videoFrame.getByLabel('Pause')).toBeVisible()
+    const iframeElement = await page.waitForSelector('iframe')
+    const Visible = await iframeElement.isVisible()
+    expect(Visible).toBe(true)
```

Comment:
the `toBeVisible` assertion can be a better fit for this use case - if the element is not visible (for example because it's loading or has entrance animation) it will wait for it to appear

https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible